nav2_follow_path_controller/src/nav2_follow_path_controller.cpp

```diff
@@ -12,66 +12,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "dwb_controller/dwb_controller.hpp"
-
 #include <chrono>
 #include <memory>
 #include <string>
 
-#include "dwb_core/exceptions.hpp"
+#include "nav2_local_planner_interface/exceptions.hpp"
 #include "nav_2d_utils/conversions.hpp"
 #include "nav2_util/node_utils.hpp"
-#include "dwb_controller/progress_checker.hpp"
+#include "nav2_follow_path_controller/progress_checker.hpp"
+#include "nav2_follow_path_controller/nav2_follow_path_controller.hpp"
 
 using namespace std::chrono_literals;
 
-namespace dwb_controller
+namespace nav2_follow_path_controller
```

Comment:
Ditto on naming - this is a bit long. Something like `nav2_control_action` to get it across - its the action that handles the controller

Reply:
I will rename the namespaces based on the package names (one we decide on that).